README.md

```diff
@@ -492,6 +492,12 @@ const config = {
      */
     dangerouslyKillFlumeWhenMigrated: false,
 
+    /**
+     * Only try to decrypt box1 messages created before this date
+     * Default: null
+     */
+    startDecryptBox1: "2022-03-25",
```

Comment:
I settled for string representation because passing in a date doesn't seem to work, it gets converted into an empty object.